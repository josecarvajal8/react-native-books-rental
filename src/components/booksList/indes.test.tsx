import React from 'react';
import renderer from 'react-test-renderer';
import BookList from '.';

test('Render booksList component correctly', ()=> {
    const bookListCmp = renderer.create(<BookList navigation={null}/>).toJSON();
    expect(bookListCmp).toMatchSnapshot();
})