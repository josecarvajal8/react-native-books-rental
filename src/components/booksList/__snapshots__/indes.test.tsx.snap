// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render booksList component correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#dee7ea",
      "flex": 1,
    }
  }
>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "height": 148.88392857142858,
        "width": 750,
      }
    }
  >
    <Image
      resizeMode="contain"
      source={
        Object {
          "testUri": "../../../src/assets/img/bc_nav_bar_3x.png",
        }
      }
      style={
        Object {
          "height": 297.76785714285717,
          "position": "absolute",
          "top": -66.99776785714286,
          "width": 753.6231884057971,
        }
      }
    />
    <View
      style={
        Object {
          "flexDirection": "row",
          "justifyContent": "space-between",
          "marginTop": 7.444196428571429,
          "paddingHorizontal": 36.231884057971016,
        }
      }
    >
      <View
        accessible={true}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "height": 29.776785714285715,
            "opacity": 1,
            "width": 36.231884057971016,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "#ffffff",
                "fontSize": 20,
              },
              undefined,
              Object {
                "fontFamily": "FontAwesome",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
      </View>
      <Text
        style={
          Object {
            "color": "#ffffff",
            "fontFamily": "Montserrat-Bold",
            "fontSize": 45.289855072463766,
            "lineHeight": 0,
          }
        }
      >
        [missing "en.titles.library" translation]
      </Text>
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "height": 29.776785714285715,
            "opacity": 1,
            "width": 36.231884057971016,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "#ffffff",
                "fontSize": 20,
              },
              undefined,
              Object {
                "fontFamily": "FontAwesome",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
      </View>
    </View>
  </RCTSafeAreaView>
  <View
    style={
      Object {
        "justifyContent": "center",
        "marginTop": 29.776785714285715,
      }
    }
  >
    <RCTScrollView
      data={Array []}
      disableVirtualization={false}
      getItem={[Function]}
      getItemCount={[Function]}
      horizontal={false}
      initialNumToRender={10}
      keyExtractor={[Function]}
      maxToRenderPerBatch={10}
      onContentSizeChange={[Function]}
      onEndReachedThreshold={2}
      onLayout={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      showsVerticalScrollIndicator={false}
      stickyHeaderIndices={Array []}
      style={
        Object {
          "height": 1161.294642857143,
          "marginBottom": 156.328125,
          "marginLeft": 63.40579710144928,
          "width": 750,
        }
      }
      updateCellsBatchingPeriod={50}
      viewabilityConfigCallbackPairs={Array []}
      windowSize={21}
    >
      <View />
    </RCTScrollView>
  </View>
</View>
`;
